packages/plugin-snowplow/src/androidMain/kotlin/ly.iterative.itly.snowplow/SnowplowPlugin.kt

```diff
@@ -41,8 +42,10 @@ actual class SnowplowPlugin actual constructor(
     override fun load(options: PluginLoadOptions) {
         logger = options.logger
         logger.debug("[plugin-snowplow] load")
+        if (config.tracker == null && config.trackerUrl == null) {
+            throw IllegalArgumentException("At least one of 'tracker' or 'trackerUrl' in SnowplowOptions is required")
+        }
         this.snowplow = config.tracker ?: Snowplow.createTracker(config.androidContext,
-                // FIXME(Dante) Throw a helpful error if there is no trackerUrl
                 "itly", config.trackerUrl ?: "", HttpMethod.POST);
```

Comment:
@dantetam if trackerUrl is know as not null at this point do we need the `?: ""`